src/controllers/ExpensesControllers.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+const { getUserById } = require('../services/userServises');
+const {
+  getAllExpenses,
+  getExpenseById,
+  deleteExpense,
+  postNewExpense,
+  updateExpense,
+} = require('./../services/expensesServices');
+
+const getExpenses = (req, res) => {
+  const { userId, categories, from, to } = req.query;
+  const expenses = getAllExpenses(userId, categories, from, to);
+
+  if (userId || categories || from || to) {
+    if (!expenses.length) {
+      res.sendStatus(404);
+
+      return;
```

Comment:
In addition to setting the status, you should also add information for the user.
```suggestion
      return res.status(404).send("No expenses found.");
```